docs/docs/integration/embedding.md

```diff
@@ -0,0 +1,328 @@
+| --- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|
+| resource | The name of the dashboard to embed                                                                                                                                                             | Yes                              |
+| user_email | The email of the user to embed the dashboard for                                                                                                                                               | No (either this or `attributes`) |
+| attributes | Json payload to be put in the access token, used to pass attributes to the dashboard for enforcing policies. When using this make sure to pass all the attributes used in your security policy | No (either this or `user_email`) |
+| ttl_seconds | The time to live for the iframe URL                                                                                                                                                            | No (Default: 86400)              |
+
+:::tip
+
+Use `"attributes": {}` for no user context.
+
+:::
+
+The response will contain an iframe URL that can be used to embed the dashboard in the customer's application. Here's an example response:
+
+```json
+{
+  "iframeSrc": "https://ui.rilldata.com/-/embed?access_token=<token>&instance_id=<id>&kind=MetricsView&resource=<dashboard-name>&runtime_host=<runtime_host>&state=&theme=",
+  "runtimeHost": "<runtime_host>",
+  "instanceId": "<id>",
+  "accessToken": "<token>",
+  "ttlSeconds": 86400
+}
+```
+
+### Frontend: Embed the dashboard
+The frontend just need to use the iframeSrc to embed the dashboard in the customer's application. Here's an example of how to embed the dashboard using iframe:
+
+```html
+<iframe title="rill-dashboard" src="<iframeSrc>" width="100%" height="1000" style="border:0;" />
+```
+
+### Example
+Here's an example of how to create a dashboard component in a React application, this component will fetch the iframe URL from the backend and embed the dashboard in the frontend:
+
+```jsx
+import React, { useEffect, useState } from 'react';
+import { useUser } from '../hooks/useUser';
+
+const RillDashboard = () => {
+  const [iframeUrl, setIframeUrl] = useState('');
+  const { userEmail } = useUser();
+
+  useEffect(() => {
+      const getIframeUrl = async () => {
+          try {
+              const response = await fetch(`/api/getIframeUrl`, {
+                  method: 'POST',
+                  headers: {
+                      'Content-Type': 'application/json',
+                  },
+                  body: JSON.stringify({
+                      resource: dashboardName,
+                      user_email: '<user-email>',
```

Comment:
The email should not be passed from the frontend (that would be a security loophole that's easy to exploit), it should be resolved on the backend based on the embedder's auth logic for resolving the current user

Reply:
yeah I think this was left by mistake, backend api already has a template for email. So I can just remove it from here.